package.json

```diff
@@ -52,7 +52,7 @@
     "pinia": "^2.0.35",
     "pinia-class": "^0.0.3",
     "regenerator-runtime": "^0.13.11",
-    "sbc-common-components": "3.0.17",
+    "sbc-common-components": "3.0.15-a",
```

Comment:
As per a conversation with Travis, this is a branch of sbc-common-components that excludes a "SBC Connect" branding commit, which wasn't ready to go to Prod. THIS version can safely go to Prod.


Ref: https://github.com/bcgov/sbc-common-components/pull/384

Yes, I created releases and looked over all the changes and did not see anything that should cause breakages in any UI.

Ref: https://github.com/bcgov/sbc-common-components/releases